src/GraphQL.NewtonsoftJson/ExecutionResultJsonConverter.cs

```diff
@@ -75,53 +82,18 @@ private void WriteErrors(ExecutionErrors errors, JsonWriter writer, JsonSerializ
                     serializer.Serialize(writer, error.Path);
                 }
 
-                WriteErrorExtensions(error, writer, serializer);
+                if (error.Extensions?.Count > 0)
+                {
+                    writer.WritePropertyName("extensions");
+                    serializer.Serialize(writer, error.Extensions);
+                } 
```

Comment:
This code within both of the serializers gets cleaned up a whole lot, just writing whatever is in the extensions property.